skimage/draw/__init__.py

```diff
@@ -1,3 +1,9 @@
+"""Utilities for generating coordinates of a bezier curve, circle, ellipse, rectangle, 
+generating ellipsoid, 
+calculating surface-area and volume of ellipsoid, 
+drawing lines, 
+setting pixel color at a coordinate.
+"""
```

Comment:
Hey @anand-pb, thanks for the first contribution! It's a good start, however, there are some formatting and style issues to improve upon. Please have a look at the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html) which also describes docstring conventions that skimage is using. 

It's a good idea to look for [examples in our library itself](https://github.com/scikit-image/scikit-image/blob/44da98e0a61f9382b0d96ad3973a1b83b2db1b5f/skimage/io/__init__.py#L1-L5).

What do you think about shortening this description to a single line?

Reply:
yes @lagru, possible.
should I be making an other pull request ?

Reply:
There is no need. :D You should be able to continue updating this pull request by pushing to your branch!

Reply:
ok, I have pushed the changes now, checks are going on

Reply:
it passed all the checks, can it be merged ?

Reply:
Once a core developer approves, in this case, yes it can be merged. If you are interested in how and when changes are merged, have a look at our [decision making guidelines](https://scikit-image.org/docs/dev/skips/1-governance.html#decision-making-process). 

I've left another suggestion you are free to ignore. After you've commented or committed the suggestion will merge this. ;)